repour/lib/scm/git.py

```diff
@@ -377,7 +377,10 @@ async def do(atomic):
             options = ["--tags"]
             failure_push_msg = "tag"
         else:
-            options = ["--follow-tags", remote, branch]
+            tags = await list_reachable_tags_from_ref(dir, branch)
+            options = [remote, branch]
+            if not tags:
+                options += ["tag"] + tags
```

Comment:
I thought this wouldn't work but I tried it on one of my test github repos and it does! :+1: